### ISSUE TITLE
Testing: bad inline actions contrast upon selected tree item

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#131500]
Refs: https://github.com/microsoft/vscode-python/issues/17040

I suggest to use white for action icons when selected item and hovered, since black on blue does not give it enough contrast. All other trees do this as far as I know.

![Screenshot 2021-08-24 at 13 46 12](https://user-images.githubusercontent.com/1926584/130610987-8704d46d-8cbc-4d67-a42b-b80fbfe795e2.png)

Here's open editors as an example

![Screenshot 2021-08-24 at 13 46 33](https://user-images.githubusercontent.com/1926584/130611079-519fbeaa-4a7b-4152-beae-0f2e34ad0ddb.png)

fyi @karthiknadig @misolori

**Comment by misolori** [comment]
@isidorn this is completely dependent on your theme, our default dark/light theme show them appropriately:

<img width="368" alt="CleanShot 2021-08-30 at 10 47 53@2x" src="https://user-images.githubusercontent.com/35271042/131382222-19ad65d5-1c64-4614-9975-3d30774cc453.png">

<img width="370" alt="CleanShot 2021-08-30 at 10 48 37@2x" src="https://user-images.githubusercontent.com/35271042/131382263-e3121bd9-43a8-4261-b25e-73e644a929fe.png">

**Comment by connor4312** [comment]
Which theme are you using @isidorn? It seems like in is globally styled by

https://github.com/microsoft/vscode/blob/77905c850e170eab00cac5ca190c7b1fe5ad43ba/src/vs/base/browser/ui/list/listWidget.ts#L791-L791

which should be applied herer

**Comment by misolori** [comment]
That looks like the [GitHub Sharp theme](https://marketplace.visualstudio.com/items?itemName=joaomoreno.github-sharp-theme) which has the same style applied in other list views:

<img width="421" alt="CleanShot 2021-08-30 at 15 33 28@2x" src="https://user-images.githubusercontent.com/35271042/131414454-51b60580-6c5b-4996-99d8-7652f1d22de3.png">

Setting the following would address this in the theme:

```json
  "workbench.colorCustomizations": {
    "[GitHub Sharp]": {
      "list.activeSelectionIconForeground": "#f0f0f0"
    },
  }
```

<img width="455" alt="CleanShot 2021-08-30 at 15 35 27@2x" src="https://user-images.githubusercontent.com/35271042/131414595-46f14b94-d749-497f-8389-c305c1ba1aa1.png">

**Comment by isidorn** [comment]
Sorry for the noise.
I will provide a PR against the theme. No need to keep this open in VS Code.

**Comment by isidorn** [comment]
Good point by @joaomoreno 
@misolori @aeschli Why isn't `activeSelectionIconForeground` inheriting from `activeSelectionForeground`?

**Comment by joaomoreno** [comment]
It feels like it should be the default.

**Comment by misolori** [comment]
The default is null because majority of themes have different active states in lists + suggest widget + quick pick and there is not a single solution that solves it for all, see https://github.com/microsoft/vscode/issues/125998 for details.
